### PR TITLE
remove travisci dzil plugin, allow for faster container builds

### PR DESCRIPTION
**Pull request by moollaza**
/cc @zachthompson @jagtalon

### CODE REVIEW

**Comment by moollaza** [comment]
Doesn't shave off much time (we're not testing nearly enough of DuckPAN's functionality!), but I'm in favor of simplifying and removing dependencies.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+sudo: false
 before_install:
   - export HARNESS_OPTIONS=j10:c HARNESS_TIMER=1
   - rm .travis.yml
```
Comment (zachthompson) [inline .travis.yml]: I don't think we need to do this now.

Comment (moollaza) [inline .travis.yml]: Right!

**Comment by moollaza** [comment]
@zachthompson this should be good now :) -- thanks for catching that.

**Comment by zachthompson** [comment]
Fantastic, off we go!

**Pull request merged**